source/options.html

```diff
@@ -82,6 +82,14 @@
 
 	</details>
 
+	<details id="export">
+		<summary><strong>🗄️ Export options</strong></summary>
```

Comment:
Export Refined-Github Settings

Reply:
> Export Refined-Github

Kinda redundant

<img width="572" alt="229799979-242c40ba-3079-48d3-8b27-11bd9001df85" src="https://user-images.githubusercontent.com/1402241/230776071-749d0add-023e-4745-9544-b0b04e0e1ffa.png">

> Settings

Chrome calls it "Options", Firefox "Preferences" and Safari "Settings" 😅 

I think it's unimportant to vary this by browser too

Reply:
I think settings makes the most sense

Reply:
> I think settings makes the most sense

That ship has long sailed:

<img width="324" alt="Screenshot 8" src="https://user-images.githubusercontent.com/1402241/230777333-4123f61d-b849-4c6d-8946-ccb24de696a1.png">

If 90% of the users see "Options", then we also call them options